pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -572,8 +573,7 @@ func getRunName(runsStatus map[string]*v1beta1.PipelineRunRunStatus, ptName, prN
 			return k
 		}
 	}
-
-	return names.SimpleNameGenerator.RestrictLengthWithRandomSuffix(fmt.Sprintf("%s-%s", prName, ptName))
+	return kmeta.ChildName(prName, fmt.Sprintf("-%s", ptName))
```

Comment:
looks like you've made the same change to `Runs` as well, please documentation for that like you've done for `TaskRuns` and `Pods`

adding that detail to the commit message and pr description would be great too!

Reply:
Nice catch, thank you. Fixed.